samples/WorkerBindingSamples/Blob/BlobInputBindingSamples.cs

```diff
@@ -0,0 +1,109 @@
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequestData req,
+            [BlobInput("input-container/sample1.txt", Connection = "Storage")] BlobClient client)
+        {
+            var downloadResult = await client.DownloadContentAsync();
+            var content = downloadResult.Value.Content.ToString();
+            _logger.LogInformation("Blob content: {content}", content);
+
+            return req.CreateResponse(HttpStatusCode.OK);
+        }
+
+        [Function(nameof(BlobInputStreamFunction))]
+        public HttpResponseData BlobInputStreamFunction(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequestData req,
+            [BlobInput("input-container/sample1.txt", Connection = "AzureWebJobsStorage")] Stream stream)
+        {
+            using var blobStreamReader = new StreamReader(stream);
+            _logger.LogInformation("Blob content: {stream}", blobStreamReader.ReadToEnd());
+
+            return req.CreateResponse(HttpStatusCode.OK);
+        }
+
+        [Function(nameof(BlobInputByteArrayFunction))]
+        public HttpResponseData BlobInputByteArrayFunction(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequestData req,
+            [BlobInput("input-container/sample1.txt", Connection = "AzureWebJobsStorage")] Byte[] data)
+        {
+            _logger.LogInformation($"Blob content: {Encoding.Default.GetString(data)}");
+            return req.CreateResponse(HttpStatusCode.OK);
+        }
+
+        [Function(nameof(BlobInputStringFunction))]
+        public HttpResponseData BlobInputStringFunction(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequestData req,
+            [BlobInput("input-container/sample1.txt", Connection = "AzureWebJobsStorage")] string data)
+        {
+            _logger.LogInformation($"Blob content: {data}");
+            return req.CreateResponse(HttpStatusCode.OK);
+        }
+
+        [Function(nameof(BlobInputBookFunction))]
+        public HttpResponseData BlobInputBookFunction(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequestData req,
+            [BlobInput("input-container/book.json", Connection = "AzureWebJobsStorage")] Book data)
+        {
+            _logger.LogInformation($"Book name: {data.Name}");
+            return req.CreateResponse(HttpStatusCode.OK);
+        }
+
+        [Function(nameof(BlobInputCollectionFunction))]
+        public HttpResponseData BlobInputCollectionFunction(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequestData req,
+            [BlobInput("input-container")] IEnumerable<BlobClient> blobs)
+        {
+            _logger.LogInformation("Blobs within container:");
+            foreach (BlobClient blob in blobs)
+            {
+                _logger.LogInformation(blob.Name);
```

Comment:
name of the blob is something the current model also get. So may be read a property which current model does not provide. Ex: `BlobContainerName`. ?